### PR TITLE
fix unicode characters in fe_nedelec_sz.h

### PR DESCRIPTION
**Pull request by davydden**
in reference to https://github.com/dealii/dealii/issues/6676#issuecomment-392307349

### CODE REVIEW

File: include/deal.II/fe/fe_nedelec_sz.h [hunk 1]
```diff
@@ -31,13 +31,13 @@ DEAL_II_NAMESPACE_OPEN
 
 /**
  * This class represents an implementation of the
- * H<sup>curl</sup>-conforming N�d�lec element described in the PhD
+ * H<sup>curl</sup>-conforming Nedelec element described in the PhD
```
[the remark this answers is not in the document]
Comment (davydden) [reply include/deal.II/fe/fe_nedelec_sz.h]: i updated.

File: include/deal.II/fe/fe_nedelec_sz.h [hunk 2]
```diff
@@ -31,13 +31,13 @@ DEAL_II_NAMESPACE_OPEN
 
 /**
  * This class represents an implementation of the
- * H<sup>curl</sup>-conforming N�d�lec element described in the PhD
- * thesis of S. Zaglmayr, <b>High Order Finite Element Methods
- * for Electromagnetic Field Computation</b>, Johannes Kepler
- * Univerit�t Linz, 2006.
+ * H<sup>curl</sup>-conforming N&eacute;d&eacute;lec element described in the
+ * PhD thesis of S. Zaglmayr, <b>High Order Finite Element Methods for
+ * Electromagnetic Field Computation</b>, Johannes Kepler Univerit&auml;t Linz,
```
Comment (bangerth) [inline include/deal.II/fe/fe_nedelec_sz.h]: while you're there, the word is also missing an 's'.

Comment (davydden) [reply include/deal.II/fe/fe_nedelec_sz.h]: Will do

**Comment by tamiko** [comment]
@davydden Roger. Thanks!

**Pull request merged**